ecl/resource_ecl_sss_approval_request_v1.go

```diff
@@ -0,0 +1,182 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceSSSApprovalRequestV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSSSApprovalRequestV1Create,
+		Read:   resourceSSSApprovalRequestV1Read,
+		Update: resourceSSSApprovalRequestV1Update,
+		Delete: resourceSSSApprovalRequestV1Delete,
+		Schema: map[string]*schema.Schema{
+			"request_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
```suggestion
				Required: true,
				ForceNew: true,
```

Reply:
Related to this.
https://github.com/nttcom/terraform-provider-ecl/pull/46#discussion_r375687948